functionary/ui/templates/unicorn/task_detail.html

```diff
@@ -1,10 +1,12 @@
 <div class="column"
-     {% if should_refresh %} unicorn:model="task" unicorn:poll-5000="refresh_task"{% endif %}>
+     {% if should_refresh %} unicorn:model="task" unicorn:poll-5000="refresh_task"{% endif %}
```

Comment:
This code is used to target partial updates to the UI instead of rendering the whole thing